### ISSUE TITLE
Local.Get with Filter returns error

### ISSUE THREAD

**Issue by etiennedi** [creativesoftwarefdn/weaviate#629]
Running the following query (copied straight from the docs)
```
{
  Local {
    Get(where: {
      operator: And,
      operands: [{
        path: ["Things", "City", "population"],
        operator: GreaterThan
        valueInt: 1000000
      }, {
        path: ["Things", "City", "inCountry", "Country", "name"],
        operator: Equal,
        valueString: "Netherlands"
      }]
    }) {
      Things {
        City {
          name
          InCountry {
            ... on Country {
              name
            }
          }
        }
      }
    }
  }
}
```

leads to this error:
```
{
  "data": {
    "Local": {
      "Get": null
    }
  },
  "errors": [
    {
      "locations": [
        {
          "column": 5,
          "line": 3
        }
      ],
      "message": "The 'path' field for the filter '{\"operator\":\"GreaterThan\",\"path\":[\"Things\",\"City\",\"population\"],\"valueInt\":1000000}' is invalid! Missing an argument after 'population'",
      "path": [
        "Local",
        "Get"
      ]
    }
  ]
```

I don't understand the error message. According to the docs (I also double-checked this with @laura-ham), the query should resolve correctly. Therefore labelling as bug.

**Comment by etiennedi** [comment]
I've emailed @moretea to help clarify what the intention of sending this error message is.

**Comment by etiennedi** [comment]
The issue is  #565. In the discussion in there there it seems like we want to keep `Thing` or `Action` in the path, but it was just out-scoped at the time because of different decision prior. If so, this bug ticket is valid? Let's maybe first wait for the output of #630 - maybe we introduce new changes there.

**Comment by etiennedi** [comment]
Agree let's not try to fix this before fixing #630. It'll most likely become obsolete because of it.

**Comment by bobvanluijt** [comment]
Please see my €0.02 in #630.
